web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -205,8 +205,40 @@ const createNewLicenses = async () => {
   console.info(`Created a total of ${newLicenses.length} licenses`);
 };
 
+const getUnUsedLicenses = async () => {
+  const localLicensesRaw = loadAllLicenses();
+  const localLicensesFiltered = localLicensesRaw.filter((it) => it.webflowId === undefined);
+  const localLicensesIds = new Set(localLicensesFiltered.map((i) => i.webflowId));
```

Comment:
Two things:

- [sand] You're using `i` here and `it` above. I'd pick a better word for both, like `contentItem`.
- [dust] `localLicensesFiltered` appears to be a list of all licenses that have no `webflowId`. But here, we're trying to create a `Set` of all the `webflowId`s of `localLicensesFiltered`. Are you sure you want to do this? Wouldn't the set just be undefined?

Reply:
Lol, don't you think the second comment in more worth of a boulder?

I changed the logic to be the opposite because apparently I can't remember consistently how filter works. Sob

Anyways I believe this should work now and I logged out all the ID's we're going to delete and did some spot checking via global search to make sure those webflow ID's don't appear locally as validation.